src/visbook/visbook.service.ts

```diff
@@ -28,4 +28,22 @@ export class VisbookService {
 
     return false;
   }
+
+  // TODO This is a suggestion to prevent exceptions getting in the way of finding a random available cabin
+  async isBookingEnabled(
+    cabinVisbookId: number,
+    checkIn: string,
+    checkOut: string,
+  ): Promise<boolean> {
+    try {
+      await this.visbookApi.getAccommodationAvailability(
+        cabinVisbookId,
+        checkIn,
+        checkOut,
+      );
+    } catch {
```

Comment:
In JS all sorts of errors can get thrown so catching all errors is a dangerous pattern IMO. For example this would also catch a syntax error.

It would be better to:
1.  Catch the error.
2. Log the error.
3. Return false.

Reply:
I used the same logger that we have already in other places. Does that make sense?

Reply:
Yep. Looks good!